core/merginapi.cpp

```diff
@@ -1181,7 +1181,7 @@ void MerginApi::createProjectFinished()
 
     CoreUtils::log( "create " + projectFullName, message );
 
-    emit projectCreated( projectFullName, false );
+    emit projectCreated( projectName, false );
 
```

Comment:
This does not sound like a good idea, the signal is used in other handlers too and we might break them this way. I would keep the `projectFullName` and parse the `projectName` out of it in your handler instead :)